components/cards/EventCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <nuxt-link
+    :to="to"
+    class="card-link"
+  >
+    <article class="event-card">
+      <div class="event-card__content">
+        <header>
+          <span class="event-card__subtitle">
+            {{ type }}
+          </span>
+          <h3 class="event-card__title">
+            {{ title }}
+          </h3>
+        </header>
+
+        <footer>
+          <div class="event-card__info-detail">
+            <Map20 class="event-card__icon-map" />
+            <span class="event-card__place">{{ place }}</span>
+          </div>
+          <div class="event-card__date-and-arrow">
+            <div class="event-card__info-detail">
+              <Calendar20 class="event-card__icon-calendar" />
+              <span class="event-card__date">{{ date }}</span>
+            </div>
+            <ArrowRight20 class="event-card__icon-arrow-right" />
+          </div>
+        </footer>
+      </div>
+
+      <div
+        class="event-card__media"
+        :style="`background-image: ${decorate(image)};`"
```

Comment:
Is there any reason why you are not just writing
```
:style="`background-image: url(${image});`"
```
and delete the decorate function? 🤔

Reply:
Also I think that if you know that the path will always start by `/images/events/` you should try something as:
```
:style="`background-image: url(/images/events/${image});`"
```

Does it make sense?

Reply:
Another solution is creating a computed property:
HTML: 
```
:style="backgroundImage"
```

JS: 
```
get backgroundImage () {
    return `background-image: url(/images/events/${this.image});`
  }
```